vite.config.js

```diff
@@ -3,5 +3,9 @@ import basicSsl from '@vitejs/plugin-basic-ssl'
 import { defineConfig } from 'vite'
 
 export default defineConfig({
-  plugins: [sveltekit(), basicSsl()]
+  plugins: [sveltekit(), basicSsl()],
+  server: {
+    // temp fix for https://github.com/sveltejs/kit/issues/11365#issuecomment-1979797510
+    proxy: {}
+  }
```

Comment:
I'm not I understand why this needed? 
Is it not working in dev mode with latest Node?

Reply:
It wasn't for me; I was getting the error in the linked thread. But if you'd prefer to leave that out, I can get rid of it :)

Reply:
Yeah, let's leave it out. Cause I'd need to review that issue